Server/package.json

```diff
@@ -1,76 +1,76 @@
 {
-  "name": "@citrineos/ocpi-server",
-  "version": "0.2.0",
-  "description": "The monitoring module for OCPP v2.0.1. This module is not intended to be used directly, but rather as a dependency for other modules.",
+  "name": "@citrineos/demo",
+  "version": "0.0.20",
```

Comment:
Is this the version number we are sticking with?

Reply:
oh no! We need to fix that, this was copied from demo, I guess this one would be @citrineos/ocpi-server and match the version of ocpi